app/src/componentLibrary/modal/modalLayout/modalLayout.jsx

```diff
@@ -36,16 +34,14 @@ export const ModalLayout = ({
   footerNode,
   okButton,
   cancelButton,
-  closeIconEventInfo,
   className,
   modalSize,
+  onCloseCallback,
```

Comment:
May be just `onClose`? More simple and clear interface. Also please add it to the readme.